app/templates/main/browse-transferring-body.html

```diff
@@ -57,286 +57,264 @@
+                                </div>
                             </div>
-                        </div>
-                        <div class="govuk-date-input__item">
-                            <div class="govuk-form-group">
-                                <input class="govuk-input govuk-date-input__input govuk-input--width-4"
-                                       id="date-from-year"
-                                       name="date-from-year"
-                                       type="text"
-                                       inputmode="numeric">
-                                <label class="govuk-label govuk-date-input__label filters-form__date__label"
-                                       for="date-from-year">(YYYY)</label>
+                            <div class="govuk-date-input__item govuk-date-input__item--browse-all">
+                                <div class="govuk-form-group">
+                                    <label class="govuk-label govuk-date-input__label filters-form__date__label"
+                                           for="date_from_year">Year</label>
+                                    <input class="govuk-input govuk-date-input__input govuk-input--width-4"
+                                           id="date_from_year"
+                                           name="date_from_year"
+                                           type="text"
+                                           value="{% if user_filters['date_range'] %}{% if user_filters['date_range']['date_from'] %}{{ user_filters['date_range']['date_from'][6:10] }}{% endif %}{% endif %}"
+                                           inputmode="numeric">
+                                </div>
                             </div>
                         </div>
-                    </div>
-                </fieldset>
-            </div>
-            <div class="govuk-form-group date-form filters-form__group filters-form__date-to-group">
-                <p class="govuk-body-m date-form browse__body filters-form__body-date">To date</p>
-                <fieldset class="govuk-fieldset" role="group" aria-describedby="date-to">
-                    <div class="govuk-date-input" id="date-to">
-                        <div class="govuk-date-input__item">
-                            <div class="govuk-form-group">
-                                <input class="govuk-input govuk-date-input__input govuk-input--width-2"
-                                       id="date-to-day"
-                                       name="date-to-day"
-                                       type="text"
-                                       inputmode="numeric">
-                                <label class="govuk-label govuk-date-input__label filters-form__date__label"
-                                       for="date-to-day">(DD)</label>
+                    </fieldset>
+                </div>
+                <div class="browse-all-filter-to-dates">
```

Comment:
can we extract the datae range component into its own html to make this easier to read and reduce duplication?

Like wise with all other shared filter components, e.g. series filter, and apply button div etc.

Reply:
I was thinking of separating both sort and filter into a component which can work for both mobile and desktop view